tests/cases/php/parentheses-useless

```diff
@@ -0,0 +1,33 @@
+---DESCRIPTION---
+Parentheses must not be used unnecessarily
+---CONTENTS---
+<?php
+
+declare(strict_types=1);
+
+$foo = ($bar !== null) ? true : false;
+$baz = ($qux) ? 1 : 0;
+$quux = (true || false) ? 0 : 1;
+
+switch ($quuz) {
+    case (true):
+    case (false):
+        return $corge / (100 + $grault);
+}
+
+---FIXED---
+<?php
+
+declare(strict_types=1);
+
+$foo = $bar !== null ? true : false;
+$baz = $qux ? 1 : 0;
+$quux = (true || false) ? 0 : 1;
+
+switch ($quuz) {
+    case true:
+    case false:
+        return $corge / (100 + $grault);
```

Comment:
I'm curious about the effect on `(60 * 60 * 24) * $days` (or similar) which has unnecessary parentheses but can make the calculation more clear

Reply:
All those examples are unaffected.

(`(60 * 60 * 24 * $days)` is also unaffected when it _is_ useless though.)